hybrasyl/Objects/VisibleObject.cs

```diff
@@ -86,13 +87,15 @@ public bool CanBeLooted(string username, out string error)
                 error = "You can't do that.";
                 return false;
             }
-            // Check if the item is part of a death pile
-            if (DeathPileTime == null) return true;
+
+            // Check if the item is part of a death pile or recent monster dropped items
+            if (ItemDropTime == null) return true;
             if (DeathPileOwner == username) return true;
-            if (DeathPileAllowedLooters.Contains(username) &&
-                (DateTime.Now - DeathPileTime.Value).Seconds > Constants.DEATHPILE_GROUP_TIMEOUT) return true;
-            if ((DateTime.Now - DeathPileTime.Value).Seconds > Constants.DEATHPILE_RANDO_TIMEOUT) return true;
+            if (ItemDropAllowedLooters.Contains(username) && (DateTime.Now - ItemDropTime.Value).TotalSeconds > Constants.DEATHPILE_GROUP_TIMEOUT) return true;
+            if ((!DeathPileOwner.Equals(string.Empty)) && (DateTime.Now - ItemDropTime.Value).TotalSeconds > Constants.DEATHPILE_RANDO_TIMEOUT) return true;
```

Comment:
No need for Equals here, either use `== string.Empty` or `string.IsNullOrEmpty(DeathPileOwner)`